src/blenderbim/blenderbim/bim/module/boundary/__init__.py

```diff
@@ -17,22 +17,27 @@
 # along with BlenderBIM Add-on.  If not, see <http://www.gnu.org/licenses/>.
 
 import bpy
-from . import ui, operator
+from . import ui, operator, prop
 
 classes = (
     operator.LoadProjectSpaceBoundaries,
     operator.LoadSpaceBoundaries,
     operator.LoadBoundary,
     operator.SelectProjectBoundaries,
     operator.ColourByRelatedBuildingElement,
+    operator.EnableEditingBoundary,
+    operator.DisableEditingBoundary,
+    operator.EditBoundaryAttributes,
     ui.BIM_PT_Boundary,
+    ui.BIM_PT_SpaceBoundaries,
     ui.BIM_PT_SceneBoundaries,
+    prop.BIMBoundaryProperties,
 )
 
 
 def register():
-    pass
+    bpy.types.Object.bim_boundary_properties = bpy.props.PointerProperty(type=prop.BIMBoundaryProperties)
```

Comment:
Please follow naming convention of BIMBoundaryProperties instead of bim_boundary_properties - sorry